features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json

```diff
@@ -818,6 +819,7 @@
   "myaccount.session.params.userIdleWarningTimeOut": 580,
   "myaccount.session.params.sessionRefreshTimeOut": 300,
   "myaccount.session.params.checkSessionInterval": 3,
+  "myaccount.personal_info.disabled_features": ["profileInfo.mobileVerification"],
```

Comment:
why do we set this as a disabled feature?

Reply:
ATM we don't have a mechanism to check if mobile verification is enabled since we cannot access governance connector configurations from myaccount side. Therefore, we have to enable mobile verification UI component via a deployment.toml config if mobile verification on update feature is enabled. Otherwise, mobile verification UI component should be is disabled in myaccount.